css/prefill.css

```diff
@@ -0,0 +1,18 @@
+.dailyQuizCreation .tags .history .todayTags button {
+    margin-right: 8px;
+    border: 1px solid;
+    border-radius: 5px;
+    padding: 4px;
+    background-color: rgb(217, 217, 217);
+}
+
+
+.dailyQuizCreation .tags .history .todayTags button:hover {
+    background-color: lightblue;
+}
+
+.dailyQuizCreation .tags .history .todayTags button:focus, 
+.dailyQuizCreation .tags .history .todayTags button.active {
```

Comment:
active can be deleted because not using for now, even though will be added in near future, but not needed for now.